python/GafferUI/TabbedContainer.py

```diff
@@ -185,6 +185,10 @@ def removeChild( self, child ) :
 		child._qtWidget().setParent( None )
 		child._applyVisibility()
 
+	def hasChild( self, child ) :
```

Comment:
This is just a duplicate of the `hasChild()` method you added earlier, at line 162.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3167/commits/4361ca224e657be1db373d6d0fe91d18a252c235